features/admin.ai.v1/components/branding-ai-loading-screen.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Box from "@oxygen-ui/react/Box";
+import CircularProgress from "@oxygen-ui/react/CircularProgress";
+import LinearProgress from "@oxygen-ui/react/LinearProgress";
+import Typography from "@oxygen-ui/react/Typography";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { ReactComponent as LoadingPlaceholder }
+    from "../../../modules/theme/src/themes/wso2is/assets/images/illustrations/ai-loading-screen-placeholder.svg";
```

Comment:
When building the app, themes are packed with the console app. Therefore, we should import the images in the theme from that packed theme.

```suggestion
import { ReactComponent as LoadingPlaceholder }
    from "../../themes/wso2is/assets/images/illustrations/ai-loading-screen-placeholder.svg";
```

Reply:
this is addressed https://github.com/wso2/identity-apps/commit/891f646b2c63094d4467060253aa82c4bd6751b5